config/environments/production.rb

```diff
@@ -74,4 +74,13 @@
   if AppConfig.environment.assets.host.present?
     config.action_controller.asset_host = AppConfig.environment.assets.host.get
   end
+  
+  config.dependency_loading = true if $rails_rake_task
+
+  require 'rack/rewrite'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.